web/src/redux/apis/__tests__/useCreateAsserionMutation.tesx.tsx

```diff
@@ -0,0 +1,26 @@
+import {act, renderHook} from '@testing-library/react-hooks';
+import fetchMock from 'jest-fetch-mock';
+import {ReduxWrapperProvider} from '../../ReduxWrapperProvider';
+import {useCreateAssertionMutation} from '../Test.api';
+
+test('useCreateAssertionMutation', async () => {
+  const testId = 22;
+  fetchMock.mockResponse(JSON.stringify({testId}));
```

Comment:
For the future, let's consider a mock factory to spawn new models to use for testing & storybook